### PR TITLE
fix to make it work on Windows 

### PR DESCRIPTION
**Pull request by h0ru5**
Fix path seperator to "/"
restores functionality for windows

### CODE REVIEW

File: areader/reader.go [hunk 1]
```diff
@@ -841,7 +841,7 @@ func (ar *Reader) assignUpdateFiles() error {
 
 // should be `headers/0000/file` format
 func getUpdateNoFromHeaderPath(path string) (int, error) {
-	split := strings.Split(path, string(os.PathSeparator))
+	split := strings.Split(path, string("/"))
```
Comment (kacf) [inline areader/reader.go:844]: Hey @h0ru5, so this is the actual fix here, correct? The rest is just stuff to make it locally work for you?

Comment (h0ru5) [reply areader/reader.go:844]: yes, correct.
 thought I forked 2 different branches to upstream and to run locally, sorry for the confusion

Comment (h0ru5) [reply areader/reader.go:844]: ok, realized, i put the PR on the wrong branch. the changhe you refer to is the only needed

**Comment by kacf** [comment]
Alright, I resubmitted it here: https://github.com/mendersoftware/mender-artifact/pull/482.

**Pull request closed without merging**